porfolio.html

```diff
@@ -16,7 +16,41 @@
             <p class="text">John Doe</p>
         </div>
     </div>
- 
+    <div class="headline">
```

Comment:
I suggest you change this to a `<section>` tag. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.